From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: USB: fix cpia_usb driver's warning messages in the syslog
Patch-mainline: in -mm, will be in 2.6.12

This fixes the cpia driver to call usb_kill_urb() instead of
usb_unlink_urb() which reduces the ammount of syslog messages when
kernel debugging is enabled.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- 1.33/drivers/media/video/cpia_usb.c	2005-03-10 23:15:09 -08:00
+++ edited/drivers/media/video/cpia_usb.c	2005-03-14 23:00:20 -08:00
@@ -440,7 +440,7 @@ static void cpia_usb_free_resources(stru
 
 	/* Unschedule all of the iso td's */
 	if (ucpia->sbuf[1].urb) {
-		usb_unlink_urb(ucpia->sbuf[1].urb);
+		usb_kill_urb(ucpia->sbuf[1].urb);
 		usb_free_urb(ucpia->sbuf[1].urb);
 		ucpia->sbuf[1].urb = NULL;
 	}
@@ -451,7 +451,7 @@ static void cpia_usb_free_resources(stru
 	}
  
 	if (ucpia->sbuf[0].urb) {
-		usb_unlink_urb(ucpia->sbuf[0].urb);
+		usb_kill_urb(ucpia->sbuf[0].urb);
 		usb_free_urb(ucpia->sbuf[0].urb);
 		ucpia->sbuf[0].urb = NULL;
 	}
